src/dialogs/more-info/ha-more-info-history.ts

```diff
@@ -70,6 +73,7 @@ export class MoreInfoHistory extends LitElement {
                 .hass=${this.hass}
                 .isLoadingData=${!this._statistics}
                 .statisticsData=${this._statistics}
+                .metadata=${this._metadata}
```

Comment:
We would fetch it and than discard it.  Now passed to the graph object instead so we don't have to do it again